packages/arb-token-bridge-ui/tests/e2e/specs/login.cy.ts

```diff
@@ -15,10 +15,10 @@ describe('Login Account', () => {
 
   before(() => {
     getInitialETHBalance(goerliRPC).then(
-      val => (l1ETHbal = formatBigNumber(val, 18, 5))
+      val => (l1ETHbal = formatAmount(val, { symbol: 'ETH' }))
```

Comment:
This update doesn't work correctly. Since your formatter already includes the symbol, the test is searching for `0.0001 ETH ETH` which it doesn't find. Please update these test cases accordingly.

![image](https://user-images.githubusercontent.com/7558499/200078414-91b667e2-5956-40b7-800d-36dd8de692a8.png)

Reply:
+ for all the failing tests ^

Reply:
Fixed